optimum/onnx/graph_transformations.py

```diff
@@ -79,3 +91,217 @@ def remove_duplicate_weights(model: ModelProto, inplace: bool = False) -> ModelP
     _remove_redundant_initializers(model, name_sharing_dict)
 
     return model
+
+
+def replace_atenops_to_gather(model: ModelProto):
+    """
+    Replaces broken ATenOp nodes back to Gather nodes.
+
+    Args:
+        model (`~onnx.ModelProto`): The ONNX model to fix.
+        inplace (`bool`, defaults to False): Whether to perform this transformation inplace.
+
+    Returns:
+        `~onnx.ModelProto`: The ONNX model fixed.
+    """
+    nodes = model.graph.node
+
+    for node in nodes:
+        if node.op_type in ["ATenOp", "ATen"]:
+            op_num = node.name.split("_")[-1]
+            new_node = onnx.helper.make_node(
+                "Gather",
+                name="Gather_" + op_num,
+                inputs=[node.input[0], node.input[1]],
+                outputs=node.output,
+            )
+
+            model.graph.node.remove(node)
+            model.graph.node.insert(int(op_num), new_node)
+
+    onnx.checker.check_model(model)
+    return model
+
+
+def _infer_output_shape(output: ValueInfoProto):
+    output_shape = []
+    for dim in output.type.tensor_type.shape.dim:
+        if getattr(dim, "dim_param"):
+            output_shape.append(getattr(dim, "dim_param"))
+        elif getattr(dim, "dim_value"):
+            output_shape.append(getattr(dim, "dim_value"))
+        else:
+            raise ValueError(f"Can not find `dim_param` nor `dim_value` in the output dimension info.")
+
+    return output_shape
+
+
+def _check_num_outputs(model1: ModelProto, model2: ModelProto):
+    if not len(model1.graph.output) == len(model2.graph.output):
+        raise ValueError(
+            f"Two model protos need to have same outputs. But one has {len(model1.graph.output)} "
+            f"outputs while the other has {len(model2.graph.output)} outputs."
+        )
+
+
+def _unify_onnx_outputs(model1: ModelProto, model2: ModelProto):
+    """
+    Unify the outputs of deux ONNX model protos. The outputs of model1 will be replaced by outputs of model2.
```

Comment:
```suggestion
    Unifies the outputs of deux ONNX model protos. The outputs of model1 will be replaced by outputs of model2.
```

Reply:
You didn't find that I wrote "two" in French :<